rom/dev/README.md

```diff
@@ -17,10 +17,8 @@
 
 ## 2. Spec Opens
 
-- Update the spec with adding certificate expiration times for vendor and owner
 - Update the firmware image format to include the runtime configuration section
 - Ability to run production signed firmware that can only run in debug mode
-- Add certificate section
```

Comment:
should line 38 below also be removed? 

"6. Certificate Formats"

Reply:
Updated.